src/plugins/twitter/twitter.js

```diff
@@ -31,6 +31,61 @@ var componentName = "wb-twitter",
 			protocol = wb.pageUrlParts.protocol;
 
 		if ( eventTarget ) {
+
+			// Ignore IE11
+			// Note: Twitter's widget script doesn't render anything in IE11... so there's no value in showing an endless loading icon in it
+			if ( !wb.ie11 ) {
+				const linkSelector = "a.twitter-timeline",
+					twitterLinks = eventTarget.querySelectorAll( linkSelector ),
+					loadingClass = "twitter-timeline-loading";
+				let loadingRemovals = 0,
+					observer;
+
+				// Insert loading icons below Twitter links
+				twitterLinks.forEach( function( twitterLink ) {
+					const loadingDiv = document.createElement( "div" );
+					loadingDiv.className = loadingClass;
+
+					twitterLink.after( loadingDiv );
+				} );
+
+				// Remove loading icon after the timeline widget appears
+				// Note: Twitter's widget script removes "a.twitter-timeline" upon filling-in the timeline's content... at which point the loading icon is no longer useful
+				observer = new MutationObserver( function( mutations ) {
```

Comment:
Comment:

What about to use the CSS animation + JS event. Like binded on this CSS selector "`.twitter-timeline-rendered`"

For an example, see wet-boew-experimental here: https://github.com/wet-boew/wet-boew-experimental/blob/main/src/wet-boew.js#L97-L101

Reply:
I think that'd be pretty cool and am considering going in that direction. It might make for cleaner/shorter code than a mutation observer. No idea which would have better performance though.

As far as the  ``.twitter-timeline-rendered`` selector goes, it looks like that gets injected by Twitter's third party script when it inserts the timeline's ``iframe`` into the page. But it still takes several seconds for the timeline to appear afterwards.

I'll probably do something along the lines of checking for the ``animationcancel`` event on ``".twitter-timeline-loading``. When Twitter's script removes the link, the loading ``div`` stops being adjacent to it - which should cause the loading icon to instantly disappear and trigger an ``animationcancel`` event.

Reply:
I experimented a bit with the``animationcancel`` event listener yesterday and it looks like it can work in this scenario :).

But it just hit me that making the plugin dependent on CSS animation events might not work as well as a mutation observer in practice :(.

Here's why:
* If a user were to use client-side CSS overrides to disable animations, it'd probably disable animation events. In that scenario, the loading icon would still visually-disappear when the Twitter link gets removed, but its ``div`` element would never get removed from the DOM.
* The ``a11y/media-prefers-reduced-motion`` rule from stylelint's ``stylelint-a11y`` plugin checks for ``@media (prefers-reduced-motion)`` overrides. If we activate that rule in the future and update WET's CSS to follow it, we'll probably want to disable the loading icon's spinning animation... which'll probably mess up the the animation event listener.